pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiMultiBrokersTest.java

```diff
@@ -122,4 +125,27 @@ public void testTopicLookup(TopicDomain topicDomain, boolean isPartition) throws
         Assert.assertEquals(lookupResultSet.size(), 1);
     }
 
+    @Test(timeOut = 30 * 1000)
+    public void testTopicGetMessageIdByTimestamp() throws Exception {
```

Comment:
this test don't verify this fix

Reply:
Hi，where is it not covered?

Reply:
you shou check the 307And404Exception also can return the response exception right?

Reply:
yes,  the requested admin address is not the Broker to which the Topic belong, will the result be successful. 
however, there is no 404 verification.